sources/packages/backend/libs/sims-db/src/entities/disbursement-feedback-errors.model.ts

```diff
@@ -42,4 +43,14 @@ export class DisbursementFeedbackErrors extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   disbursementSchedule: DisbursementSchedule;
+
+  /**
+   * E-Cert feedback error received.
+   */
+  @ManyToOne(() => ECertFeedbackError, { eager: false, cascade: false })
+  @JoinColumn({
+    name: "ecert_feedback_error_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  ecertFeedbackError: ECertFeedbackError;
```

Comment:
Adding it here also because this would be the principal one.
https://github.com/bcgov/SIMS/pull/3212#discussion_r1588538151